src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    operate(callback, operand) {
+      return callback(operand);
+    },
+
+    reset() {
+      obj.result = 0;
```

Comment:
it should be 'this' everywhere instead of obj. Technically this will be equal to obj, but still it is less confusing